What is it like being a teacher page (#4405)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/teaching-as-a-career/what-its-like-to-be-a-teacher.md

---
title: What it's like to be a teacher
subject: chemistry
title_paragraph: |-

>  <p>Teaching is a challenging but rewarding career that allows you to make a significant impact on pupils' lives. With no two days being the same, it’s an excellent choice for those who thrive in a dynamic and varied work environment.</p>

Reviewer: ```suggestion
 <p>Teaching is a challenging but rewarding career that allows you to make a significant impact on pupils' lives. With no 2 days being the same, it’s an excellent choice for those who thrive in a dynamic and varied work environment.</p>
```

 
description: |-

>     Find out what it's like to be a teacher including the sort of things you'll do day to day.

Reviewer: ```suggestion
    Find out what it's like to be a teacher including the kind of things you'll do each day.
```

Reviewer: Changed this as wasn't sure if it should be day-to-day or day to day!

layout: "layouts/minimal"
colour: pastel yellow-yellow
image:  "static/images/content/hero-images/geography.jpg"
keywords:
  - a day in the life of a teacher 

>   - challeneges of teaching 

Reviewer 2: ```suggestion
  - challenges of teaching 
```
